alchemy/src/cloudflare/wrangler.json.ts

```diff
@@ -557,7 +571,6 @@ function processBindings(
     } else if (binding.type === "json") {
       // TODO(sam): anything to do here? not sure wrangler.json supports this
     } else if (binding.type === "secrets_store") {
-      // Secrets Store bindings are handled via secrets_store_secrets array in wrangler.json
```

Comment:
you need to add them to the spec. we are generating wrangler.json, why are you not generating the correct wrangler.json?